test/e2e/framework/test_context.go

```diff
@@ -354,7 +361,10 @@ func RegisterCommonFlags(flags *flag.FlagSet) {
 
 	flags.StringVar(&TestContext.ProgressReportURL, "progress-report-url", "", "The URL to POST progress updates to as the suite runs to assist in aiding integrations. If empty, no messages sent.")
 	flags.StringVar(&TestContext.SpecSummaryOutput, "spec-dump", "", "The file to dump all ginkgo.SpecSummary to after tests run. If empty, no objects are saved/printed.")
-	flags.StringVar(&TestContext.DockerConfigFile, "docker-config-file", "", "A file that contains credentials which can be used to pull images from certain private registries, needed for a test.")
+	flags.StringVar(&TestContext.DockerConfigFile, "docker-config-file", "", "A docker credential file which contains authorization token that is used to perform image pull tests from an authenticated registry. For more details regarding the content of the file refer https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#log-in-to-docker-hub")
```

Comment:
I think we agreed to leave the `--docker-config-file` flag as it is so the current users don't get impacted, and instead introduce a new flag for setting private registry credentials.

Reply:
But those two now set the same field. So if someone uses `-e2e-docker-config-file`, they might as well just use `-docker-config-file` because it has the exact same effect?

Reply:
The --docker-config-file is being used for an image pull test from an authenticated registry here - https://github.com/kubernetes/kubernetes/blob/d004f324d3f5dc92f0baf2e6e176877e05471ab9/test/e2e/common/node/runtime.go#L293 
whereas the --e2e-docker-config-file is being used to pass in the registry credentials so that it can be utilized by all the spawned e2e pods whose third-party images reside in an authenticated registry.

Reply:
If the goal is to have two different flags with different meanings, then you need two separate fields in TestContext. Otherwise both flags have the exact same effect.

Reply:
I'm using the field `E2EDockerConfigFile` for `--e2e-docker-config-file` flag https://github.com/kubernetes/kubernetes/blob/abba87da2e9b6b0bc33ec62d9f93aaac98edaa15/test/e2e/framework/test_context.go#L366

Reply:
You are right of course. Not sure why I missed that, sorry.

Reply:
no issues at all